automation/taskcluster/lib/tasks.py

```diff
@@ -491,7 +484,7 @@ def _craft_raptor_task(
             _DEFAULT_TASK_URL, signing_task_id, DEFAULT_APK_ARTIFACT_LOCATION
         )
         architecture, _ = get_architecture_and_build_type_from_variant(variant)
-        worker_type = 'gecko-t-ap-perf-p2' if force_run_on_64_bit_device or architecture == 'aarch64' else 'gecko-t-ap-perf-g5'
```

Comment:
Hm, am not familiar with these workerTypes but out of curiosity, are we to retire https://tools.taskcluster.net/provisioners/proj-autophone/worker-types/gecko-t-ap-perf-p2 ?

Reply:
Yeah, I snuck the worker-type change as part of this PR as requested by [this thread](https://mozilla.slack.com/messages/CA4QR1M2R/convo/CA4QR1M2R-1557352751.433000/)